README.md

```diff
@@ -86,28 +61,28 @@ MONIKER="$(hostname)"
 palomad init "$MONIKER"
 ```
 
-Copy several files from the original server to the new server:
-  - `.paloma/config/app.toml`
-  - `.paloma/config/client.toml`
-  - `.paloma/config/config.toml`
-  - `.paloma/config/genesis.json`
+Copy the `genesis.json` file of the testnet we wish to connect to
+
+```shell
+wget -O .paloma/config/genesis.json https://raw.githubusercontent.com/palomachain/testnet/master/columba/genesis.json
```

Comment:
@measure-fi could we pick another name which doesn't sound like it was taken directly from terra? :)

Reply:
Renamed to devnet.